python/swiftnav/almanac.pxd

```diff
@@ -7,7 +7,27 @@
 # EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
 # WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
 
```

Comment:
I find it helpful to have short descriptions about what a module contains, not sure if something similar is found elsewhere in the supporting code for this, but it could be beneficial in these modules as well.